file_util/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+find_package(Boost 1.54 REQUIRED COMPONENTS filesystem)
```

Comment:
Boost.Filesystem depends on Boost.System. I'd append `system` after `filesystem`, and use `${Boost_LIBRARIES}' for `add_leatherman_deps`.